src/ixdat/db.py

```diff
@@ -418,15 +418,21 @@ def load_data(self, db=None):
 class PlaceHolderObject:
     """A tool for ixdat's laziness, instances sit in for Saveable objects."""
 
-    def __init__(self, i, cls, backend=None):
+    def __init__(self, identity, cls, backend=None):
         """Initiate a PlaceHolderObject with info for loading the real obj when needed
 
         Args:
-            i (int): The id (principle key) of the object represented
+            identity (int or tuple): The id (principle key) of the object represented OR
+                the short identity, i.e. a tuple of the id and the backend. In the later
+                case, identity[1] overrides a backend if given
             cls (class): Class inheriting from Saveable and thus specifiying the table
             backend (Backend, optional): by default, placeholders objects must live in
                 the active backend. This is the case if loaded with get().
         """
+        if isinstance(identity, int):
+            i = identity
+        else:
+            backend, i = identity
```

Comment:
`identity` is an `int` if the backend is the active backend. Otherwise, the backend has to be specified to fully identify the object, and so `identity` becomes a tuple. I agree it is not ideal. Let's fix it elsewhere (#75).